examples/common/pigweed/rpc_services/Device.h

```diff
@@ -418,6 +419,14 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
         {
             mCommissionableDataProvider.SetSpake2pVerifier(ByteSpan(request.verifier.bytes, request.verifier.size));
         }
+
+        // Resetart commissioning window to recache the value:
+        {
+            DeviceLayer::StackLock lock;
+            Server::GetInstance().GetCommissioningWindowManager().CloseCommissioningWindow();
+            Server::GetInstance().GetCommissioningWindowManager().OpenBasicCommissioningWindow();
```

Comment:
```suggestion
            {
                DeviceLayer::StackLock lock;
                Server::GetInstance().GetCommissioningWindowManager().CloseCommissioningWindow();
            }
            // Let other tasks possibly work since Commissioning window close/open are "heavy"
            {
                Server::GetInstance().GetCommissioningWindowManager().OpenBasicCommissioningWindow();
            }
```

Reply:
Done